custom-vu/src/main/kotlin/jces1209/vu/page/DcIssueNavigator.kt

```diff
@@ -0,0 +1,30 @@
+package jces1209.vu.page
+
+import org.openqa.selenium.By
+import org.openqa.selenium.WebDriver
+import org.openqa.selenium.support.ui.ExpectedConditions.*
+import java.time.Duration
+
+class DcIssueNavigator(
+    driver: WebDriver
+) {
+    private val page = FalliblePage.Builder(
+        driver,
+        and(
+            or(
+                presenceOfElementLocated(By.cssSelector("ol.issue-list")),
+                presenceOfElementLocated(By.id("issuetable")),
+                presenceOfElementLocated(By.id("issue-content"))
+            ),
+            presenceOfElementLocated(By.id("key-val")),
+            presenceOfElementLocated(By.className("issue-body-content"))
+        )
+    )
+        .cloudErrors()
```

Comment:
`cloudErrors` in DC page object? Perhaps you meant Server errors?

- [ ] detect DC errors